edc-controlplane/edc-controlplane-postgresql-hashicorp-vault/notice.md

```diff
@@ -2,24 +2,24 @@
 
 An EDC Control Plane using PostgreSQL as persistence backend, and HashiCorp Vault as secret store.
 
-DockerHub: https://hub.docker.com/r/tractusx/edc-controlplane-postgresql-hashicorp-vault
+DockerHub: <https://hub.docker.com/r/tractusx/edc-controlplane-postgresql-hashicorp-vault>
```

Comment:
I'm assuming these changes are required by docasaurus? If so, please note this in the PR description.

Reply:
Indirectly. Markdownlint (and apparently some parsers) doesn't like bare links.

Reply:
Mentioned it in the description, as requested.